src/Worker.ts

```diff
@@ -10,7 +10,7 @@ export function startNodeWorker(args) {
   let { host, port, workerId, requestId, grpcMaxMessageLength } = parseArgs(args.slice(2));
   if (!host || !port || !workerId || !requestId || !grpcMaxMessageLength) {
     console.log('usage --host hostName --port portNumber --workerId workerId --requestId requestId --grpcMaxMessageLength grpcMaxMessageLength');
-    throw new Error('Connection info missing');
+    throw new Error('gRPC client connection info is missing or null. Check \'hostName\', \'portNumber\', \'workerId\', \'requestId\', and \'grpcMaxMessageLength\'');
```

Comment:
Could you split these out to be individual validations so it's clear which is missing or incorrect?

My issue was `grpcMaxMessageLength` was 0. It confused me b/c it's a blanket check so it wasn't specific (and technically it wasn't missing or null). So I tested it locally for a bit until I realized that it was the culprit.